### PR TITLE
speedDevelopers Pull Request

### PR DESCRIPTION
**Pull request by pczern**

### CODE REVIEW

File: server/index.html [hunk 1]
```diff
@@ -1,23 +1,53 @@
 <!DOCTYPE html>
 <html>
     <head>
+          <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
         <title>Montreus Chat</title>
+        <link rel="stylesheet" type="text/css" href="reset.css"> 
+       <link href="http://cdn.foundation5.zurb.com/foundation.css" rel="stylesheet" type="text/css"> 
+<<<<<<< HEAD
```
Comment (thomas-daniels) [inline server/index.html]: Looks like there are some unresolved merge conflicts.

File: server/index.html [hunk 2]
```diff
@@ -136,6 +598,21 @@ <h1 style="width:100%; text-align: center; color: white; font-size: 22px; font-w
                                 $(".connectionLabel").html(userList);
                                       }
                             });
+                                $(function() {
```
Comment (gtomitsuka) [inline server/index.html]: $(document).ready(function{}); is the same as $(function(){});. There shouldn't be a ready event inside the ready event.

**Comment by gtomitsuka** [comment]
But well it looks pretty good for me, actually.

**Comment by gtomitsuka** [comment]
This will also fix #5 by Unihedron.

**Comment by gtomitsuka** [comment]
Thanks! Seems good enough for me. I'll merge it

**Comment by pczern** [comment]
There is probably a problem with resizing in the old version, i made a new
pull request

2015-02-07 10:32 GMT+01:00 Gabriel Tomitsuka notifications@github.com:

> Merged #13 https://github.com/gtomitsuka/montreus-chat/pull/13.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/gtomitsuka/montreus-chat/pull/13#event-231924825.

## 

http://de.linkedin.com/pub/philipp-czernitzki/94/658/84
Google+ https://plus.google.com/u/0/101073241749567186183/posts/p/pub
Twitter https://twitter.com/speedDeveloper
Webseite: API-Developers http://api-developers.com

**Pull request merged**